### PR TITLE
feat(setup_devconsole):Toggle feature flag by fetching from devconsole CRD

### PR DESCRIPTION
**Pull request by karthikjeeyar**
Set feature flags based on the gitsource CRD.

### CODE REVIEW

File: frontend/public/features.ts [hunk 1]
```diff
@@ -191,9 +191,16 @@ const detectUser = dispatch => coFetchJSON('api/kubernetes/apis/user.openshift.i
     },
   );
 
+const devopsConsolePath = `${k8sBasePath}/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/gitsources.devopsconsole.openshift.io`;
 const detectDevConsole = dispatch => {
   const devconsoleEnabled = !!(process.env.DEVCONSOLE_ENABLED && process.env.NODE_ENV === 'development');
-  setFlag(dispatch, FLAGS.SHOW_DEV_CONSOLE, devconsoleEnabled);
+  coFetchJSON(devopsConsolePath)
```
Comment (christianvogt) [inline frontend/public/features.ts:199]: You can avoid the fetch if `devconsoleEnabled` is already `false`.

**Comment by karthikjeeyar** [comment]
@christianvogt Added the Operators installation script and updated the CRD checks.

**Comment by christianvogt** [comment]
When minishift isn't running, the script takes a very long time to report any feedback and even though the first command isn't successful, the rest of the commands continue to run.

Maybe there's not much you can do here as it's a result of the `oc` tool.

File: frontend/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "dev": "rm -rf ./public/dist/ && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
-    "devconsole": "rm -rf ./public/dist/ && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --config webpack.devconsole.config.ts --env.devconsole=true --watch --progress",
+    "devconsole": "sh devconsole.sh && rm -rf ./public/dist/ && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --config webpack.devconsole.config.ts --env.devconsole=true --watch --progress",
```
Comment (christianvogt) [inline frontend/package.json]: We no longer need `webpack.devconsole.config.ts`, correct?

Comment (karthikjeeyar) [reply frontend/package.json]: @christianvogt Yes , we no longer need it. I have removed and pushed the changes

**Review by christianvogt — APPROVED** [review]

**Pull request merged**